src/core/1/guides/kuzzle-depth/scalability/index.md

```diff
@@ -6,12 +6,38 @@ title: Scalability
 
 # Scalability
 
-Kuzzle can scale horizontally, provided our [official Cluster Plugin](https://github.com/kuzzleio/kuzzle-plugin-cluster) is installed.
+Kuzzle can scale horizontally, provided our [official Cluster Plugin](https://github.com/kuzzleio/kuzzle-plugin-cluster) is installed.  
 
-This guide covers how clustering capabilities can be added to Kuzzle.
+This guide covers Kuzzle Cluster features and how clustering capabilities can be added to Kuzzle.
 
 ---
 
+## Kuzzle Cluster features
+
+Kuzzle uses a cluster in [masterless mode](https://www.wikiwand.com/en/Shared-nothing_architecture) to ensure maximum resilience.  
```

Comment:
```suggestion
Kuzzle uses a cluster in [masterless mode](https://en.wikipedia.org/wiki/Shared-nothing_architecture) to ensure maximum resilience.  
```

Reply:
Why ? Wikiwand is just a better version of wikipedia